File: LeaguerManagement.UI/src/app/models/authentication/authentication-user.model.ts
import {LoggedUserModel} from '@app/models/authentication/logged-user.model';

export class AuthenticationUserModel {
  accessToken: string;
  imageToken: string;
  refreshToken: string;
  exp: number;
  id: number;
  user: LoggedUserModel;

  public constructor(init?: Partial<AuthenticationUserModel>) {
    Object.assign(this, init);
  }
}

File: LeaguerManagement.Entities/ViewModels/Authentication/AuthenticationUserModel.cs
namespace LeaguerManagement.Entities.ViewModels
{
    public class AuthenticationUserModel
    {
        public string AccessToken { get; set; }
        public string ImageToken { get; set; }
        public string RefreshToken { get; set; }
        public long? Exp { get; set; }
        public string Id { get; set; }

        public LoggedUserModel User { get; set; }
    }
}
